spec/knapsack_pro/config/temp_files_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe KnapsackPro::Config::TempFiles, :clear_tmp do
+  describe 'TEMP_DIRECTORY_PATH' do
+    subject { described_class::TEMP_DIRECTORY_PATH }
+
+    it 'returns temporary directory path' do
+      expect(subject).to eq '.knapsack_pro'
+    end
+  end
+
+  describe '.ensure_temp_directory_exists!' do
+    let(:gitignore_file_path) { '.knapsack_pro/.gitignore' }
+
+    subject { described_class.ensure_temp_directory_exists! }
+
+    it 'creates .gitignore file' do
+      expect(File.exist?(gitignore_file_path)).to be false
+      subject
+      expect(File.exist?(gitignore_file_path)).to be true
+    end
+
+    it '.gitignore file has content' do
+      subject
+
+      expect(File.read(gitignore_file_path)).to include '# This directory is used by knapsack_pro gem for storing temporary files during tests runtime'
+      expect(File.read(gitignore_file_path)).to include '# Ignore all files, and do not commit this directory into your repository.'
+      expect(File.read(gitignore_file_path)).to include '# Learn more at https://knapsackpro.com'
+      expect(File.read(gitignore_file_path)).to include '*'
```

Comment:
Can we test it in a way that will ensure this is a whole line containing just the asterisk? Currently as I understand it, any asterisk anywhere in the file would make this expectation pass. Business-wise we need more than that.

And if we are testing for the line with the asterisk, then maybe we could improve the example description to reflect that (maybe just updating it so that it mentions "_correct_ content" or sth like that is good enough?)

BTW, I'd be fine with removing all of the other expectations (checking for the commented out content), as I don't think they add much.